highcharts/modules/no-data-to-display.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for Highcharts No Data to Display
+// Type definitions for Highcharts No Data to Display 4.2.7 
 // Project: http://www.highcharts.com/
 // Definitions by: Andrey Zolotin <http://github.com/nitoloz>
+// Definitions byt: Rowell Heria <http://github.com/rowellx68>
```

Comment:
Use a single line of authors: `// Definitions by: Andrey Zolotin <http://github.com/nitoloz>, Rowell Heria <http://github.com/rowellx68>`